staging/config/unclassified.yaml

```diff
@@ -1,5 +1,14 @@
 unclassified:
   gitHubPluginConfig:
   jiraGlobalConfiguration:
+    sites:
+      - credentialsId: ""
```

Comment:
maybe we can even remove this whole line?

Reply:
i will try this

`jiraGlobalConfiguration:
    sites:
      - url: "https://jira-preprod.nuxeo.com/"`